src/website/app/demos/ThemeProvider/index.js

```diff
@@ -15,13 +15,17 @@
  */
 
 /* @flow */
+import bestPractices from './bestPractices';
 import examples from './examples';
 
 const doc = require('!!react-docgen-loader!../../../../ThemeProvider/ThemeProvider');
 
 export default {
+  bestPractices,
   doc,
   examples,
   slug: 'theme-provider',
-  title: 'ThemeProvider'
+  title: 'ThemeProvider',
+  whenHowToUse: `ThemeProvider can be used as a wrapper around the root App component to override theme variables.
+It can also be wrapped around individual React instances for more granular control.`
```

Comment:
"It can also be used deeper within your app, for theming specific areas."?

Granular control for individual React instances is provided by `createThemedComponent`.